src/components/Stock.js

```diff
@@ -1,15 +1,22 @@
-import React from "react";
+import React from "react"
+
+function Stock({ stock, handleBuy, handleDelete }) {
```

Comment:
Instead of having two props that handle click, you can reduce it to one prop, maybe called `handleClick`. Then, when you render this component, just pass either the `handleBuy` or the `handleDelete` function (from the parent) to `handleClick`. This would eliminate the need for the conditionally a bit further down.

Reply:
I talked to TJ about this yesterday afternoon, makes much more sense!

Reply:
Awesome!